.gitpod.yml

```diff
@@ -21,9 +21,12 @@ tasks:
       nvm use default
     command: |
       pnpm start
-  - name: Run backend
+      env:
+          NUXT_TELEMETRY_DISABLED: '0'
```

Comment:
This needs to be in the setup task area and currently the indention is wrong

Reply:
@anbraten done here cba95a6583520243450e95ceac44978ec25bf17c